### PR TITLE
(130) Added InChannelContract.SupplierCustomerId

### PR DESCRIPTION
**Pull request by shilpa-cronj**
#130 - Add InChannelContract.SupplierCustomerId

### CODE REVIEW

File: src/server/db/migrations/6-InChannelContract.main.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+'use strict'
+ const DataTypes = require('sequelize');
+ module.exports.up = (db,config)=> {
+ 	return db.queryInterface.addColumn(
+ 		'InChannelContract',
+ 		'SupplierCustomerId', {
```
Comment (nczeranka) [inline src/server/db/migrations/6-InChannelContract.main.js]: We use camel case ids with lowercase char at the beginning ( =>supplierCustomerId)

File: docker-compose.yml [hunk 2]
```diff
@@ -32,7 +32,7 @@ services:
       NODE_ENV: 'development'
     ports:
      - '3007:3007'
-
+    command: npm run dev
```
[the remark this answers is not in the document]
Comment (shilpa-cronj) [reply docker-compose.yml]: had added that for some debugging that i was doing
will remove

File: Dockerfile [hunk 3]
```diff
@@ -1,13 +1,26 @@
...
+ENV NODE_ENV=development NODE_PATH=/var/tmp/base/node_modules PATH=${PATH}:${NODE_PATH}/.bin
+RUN yarn
+
+WORKDIR /home/node/einvoice-send
+
+# Bundle app source by overwriting all WORKDIR content.
+COPY . tmp
+
+# Change owner since COPY/ADD assignes UID/GID 0 to all copied content.
+RUN chown -Rf node:node tmp; rsync -a tmp/ ./ && rm -rf tmp
```
Comment (nczeranka) [inline Dockerfile]: Please do not change the build process unless it is discussed.

File: package.json [hunk 4]
```diff
@@ -67,7 +67,7 @@
   "scripts": {
     "start": "node src/server/index.js",
     "dev": "npm run setupConsul ; npx nodemon --watch src/server src/server/index.js",
-    "test": "npm run clean ; npm run setupConsul ; npx nyc ./node_modules/.bin/mocha --recursive -R mocha-junit-reporter",
+    "test": "npm run clean ; npm run setupConsul ; npx nyc ${NODE_PATH}/.bin/mocha --recursive -R mocha-junit-reporter",
```
Comment (nczeranka) [inline package.json]: Please do not change the build process unless it is discussed.

**Pull request merged**